plenum/common/roles.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class Roles(Enum):
+    TRUSTEE = "0"
+    TGB = "1"
```

Comment:
Discussed with Lovesh:
it was discussed with Jason sometime, and approved by him. An example why we need it: sending POOL_UPGRADE txn.